builtin/settingtypes.txt

```diff
@@ -1760,6 +1760,9 @@ deprecated_lua_api_handling (Deprecated Lua API handling) enum log none,log,erro
 #    Enable random user input (only used for testing).
 random_input (Random input) bool false
 
+#    Enable random mod loading (mainly used for testing).
+random_mod_load_order (Random mod load order) bool false
```

Comment:
I would probably prefer a single more general setting in case we want to randomize more things in the future. (I'm not sure what I'd call that setting though, or whether other devs would agree.)